sdk/ai/Azure.AI.Inference/src/Customized/ChatCompletionsClient.cs

```diff
@@ -133,15 +156,22 @@ public virtual async Task<StreamingResponse<StreamingChatCompletionsUpdate>> Com
                     baseResponse,
                     (responseForEnumeration)
                         => SseAsyncEnumerator<StreamingChatCompletionsUpdate>.EnumerateFromSseStream(
+                            otel,
```

Comment:
NIT: new parameters generally should be added to the end, and possibly made optional. Related to my question above about how this all works for someone who isn't trying to use the feature

Reply:
Fixed